FindTown/FindTownUI/Sources/FindTownUI/Button/RatingRadioButton.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  File.swift
+//  
+//
+//  Created by 김성훈 on 2022/12/15.
+//
+
+import UIKit
+
+public final class RatingRadioButton: UIView {
+    
+    public enum StarType {
+        case none
+        case three
+        case five
+    }
+    
+    private var starType: StarType
+    private var labelPadding: Double = 10.0
+    
+    private let backgroundView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    public let radioButton: RadioButton = {
+        let btn = RadioButton()
+        btn.isUserInteractionEnabled = false
+        return btn
+    }()
+    
+    private lazy var starStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.distribution = .fill
+        stackView.spacing = 5
+        stackView.isUserInteractionEnabled = false
+        return stackView
+    }()
+    
+    private let star1: StarButton = {
+        let btn = StarButton()
+        btn.isUserInteractionEnabled = false
+        return btn
+    }()
+    
+    private let star2: StarButton = {
+        let btn = StarButton()
+        btn.isUserInteractionEnabled = false
+        return btn
+    }()
+    
+    private let star3: StarButton = {
+        let btn = StarButton()
+        btn.isUserInteractionEnabled = false
+        return btn
+    }()
+    
+    private let star4: StarButton = {
+        let btn = StarButton()
+        btn.isUserInteractionEnabled = false
+        return btn
+    }()
+    
+    private let star5: StarButton = {
+        let btn = StarButton()
+        btn.isUserInteractionEnabled = false
+        return btn
+    }()
+    
+    private let rattingGuirdTitle: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .gray
+        return label
+    }()
+    
+    public init(starType: StarType) {
+        self.starType = starType
+        super.init(frame: CGRect.zero)
+        
+        addView()
+        setLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func addView() {
+        starStackView.addArrangedSubview(star1)
+        starStackView.addArrangedSubview(star2)
+        starStackView.addArrangedSubview(star3)
+        starStackView.addArrangedSubview(star4)
+        starStackView.addArrangedSubview(star5)
```

Comment:
```suggestion

        let starButtons = [star1, star2, star3, star4, star5]
        
        starButtons.forEach {
            starStackView.addArrangedSubview($0)
        }
        
```

요렇게 버튼을 묶어서 작성하는건 어떠세요?... 아래 switch문의 none과 five 케이스에서도 사용할 수 있을 것 같아 제안드려봅니당☺️..!

Reply:
오오.. 좋습니다! 최종 수정하고 다시 PR 날리도록 하겠습니다!! :)